Project.toml

```
@@ -7,6 +7,7 @@ version = "0.1.0"
 MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
 
 [compat]
+julia = "1.5"
```

```suggestion
julia = "1.0"
```
There's nothing in here specific to later versions of julia. Our CI is run against julia 1.0, and i think we can support it! :)